Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10.8
+FROM python:3.11
```

Comment:
installing some of the requirements errors for me with python 3.11:
```
× Encountered error while trying to install package.
╰─> lru-dict
```
[This post](https://stackoverflow.com/questions/75894242/error-when-trying-to-install-pip3-install-lru-dict) seems to imply that it is indeed due to the python version.

Reply:
For some reason the docker image builds fine though :/
Will use that for testing.